experiment/kind-detect-local-e2e.sh

```diff
@@ -176,6 +176,8 @@ ${kubelet_extra_args}
   ---
   kind: KubeProxyConfiguration
   detectLocalMode: ${KUBE_PROXY_DETECT_LOCAL_MODE:-ClusterCIDR}
+  detectLocal:
+    interfacePrefix: veth # used only with detectLocalMode "InterfacePrefix"
```

Comment:
TODO: change this to interfaceNamePrefix based on https://github.com/kubernetes/kubernetes/pull/95400#discussion_r834711681